### DISCUSSION TITLE
Annoying and false trend arrows using COMPANION APP

### DISCUSSION THREAD

**Discussion by pablodr350** [NightscoutFoundation/xDrip#3766] [Bug reports]
Hi, really the trend arrows is an issue not solved never (ie, using DEXCOM APP and broadcasting data to xdrip) where dexcom app shows ie a down arrow, and xdrip 2 down arrows....

actually stopped to use dexcom app in this way, to start to use COMPAION APP source.
ABSOLUTELY WORSE¡¡¡¡ you can get a real lateral arrow in CamAPS and 2 arrows down in xdrip.
Actually I use xdrip for alarms, but the trend arrows are literally unusuable.

why xdrip, as source, doesn´t get data (as do) and trend arrow from CAMAPS and show it? no sense to show a false valor continuosly.

is it possible, in pebble watch, disable the trend arrows showed, and leave only the value? since is a real issue and you can get stressed since always (except horizontal arrow) is wrong.

thanks

**Comment by Navid200** [comment]
Have you enabled the `Companion Bluetooth` setting?
https://navid200.github.io/xDrip/docs/Follow/CompanionApp.html

Can you show a screenshot and explain what you expect it to show instead?

**Reply by pablodr350** [reply]
Hi, not, I hadn´t it active. I did it but same result.

basically, this is happen (as was happening previously with Dexcom app and Xdrip using collector app).

value is always correct, but arrows are crazy (in xdrip of course, Camaps is fine.)

maybe isn´t possible to solve this, perfect would be to catch the camAps app trends and use them (since xdrip here doesn´t need to "think" any, only read and show camaps info)

regards

![Screenshot_20241111_180306_One UI Home](https://github.com/user-attachments/assets/684d6258-b236-44a3-9942-f18257257f2f)
![Screenshot_20241111_190424_One UI Home](https://github.com/user-attachments/assets/35da2210-c290-410d-93f8-86d6cebd10d2)

![Screenshot_20241111_093440_One UI Home](https://github.com/user-attachments/assets/382a68b8-eccf-4ec1-8820-db202db008d0)
![Screenshot_20241111_103602_One UI Home](https://github.com/user-attachments/assets/3bc7b335-b8df-41dc-81e5-17a1710f104a)
![Screenshot_20241111_110441_One UI Home](https://github.com/user-attachments/assets/2008caa4-1204-473a-9761-2bfc769790a3)
![Screenshot_20241111_111046_One UI Home](https://github.com/user-attachments/assets/818815f3-366b-442b-8abf-e19926719e46)
![Screenshot_20241111_113449_One UI Home](https://github.com/user-attachments/assets/637c609f-4633-4e6f-8792-ec0f91474cd2)

**Reply by Navid200** [reply]
We may some day be able to also get the trend arrow from the companion app.  Currently, we cannot.

Would it help if you could hide the trend arrow considering you find it annoying?

**Comment by pablodr350** [comment]
yes, option to remove the "crazy" trend arrow (with companion app or dexcom app, it was happening too, not as hard) would be fine.
I don´t like to see in my watch (pebble) a false trend arrow.

other option, I´m afraid, would be add a "soft" arrow detector, meaning decreasing the trend sensivithy to companion values.

**Reply by Navid200** [reply]
I can add an option to remove the trend arrow for xDrip.  But, this has nothing to do with WearOS.  I have no experience with WearOS.

**Reply by pablodr350** [reply]
thanks¡ an option to disable trend arrows would be fine, at least to use with companion.

Maybe (i´m using old pebble, not wearos) if doesn´t show in pebble too if don´t get value for it, but knowing that trend is "false" is enough in phone to remove it.
